src/odb/include/odb/db.h

```diff
@@ -7642,22 +7659,110 @@ class dbPowerSwitch : public dbObject
 class dbScanChain : public dbObject
 {
  public:
+  dbSet<dbScanPartition> getScanPartitions() const;
+
+  dbSet<dbScanList> getScanLists() const;
```

Comment:
How do lists and and partitions interact?  Can a partition span multiple lists?  Include part of a list?

Reply:
> Can a partition span multiple lists?

Yes but not partial lists. Let's say that I have a mux -> SFF1 -> SFF2. I need to express that like [mux, SFF1], [SFF2]. If they are of the same clock and edge then they will be in the same partition.

I changed this, It think this model handles better what we want to express here:

scan chain -> scan_partition -> scan_list -> scan_inst

Wdyt?